pkg/services/ngalert/ngalert.go

```diff
@@ -396,6 +396,28 @@ func configureHistorianBackend(ctx context.Context, cfg setting.UnifiedAlertingS
 	}
 
 	met.Info.WithLabelValues(cfg.Backend).Set(1)
+	if cfg.Backend == "fanout" {
```

Comment:
I think we should make it enum and validate for supported values during the config parsing. Also, I think we should make it case-insensitive

Reply:
Good idea! I also just realized the `fanout_secondaries` key also trims whitespace, but none of the others do this.

I'll do all 3 - enum, normalize, and trim.